monkey/infection_monkey/Pipfile

```diff
@@ -5,13 +5,12 @@ name = "pypi"
 
 [packages]
 cryptography = "==2.5" # We can't build 32bit ubuntu12 binary with newer versions of cryptography
-pyinstaller = {git = "git://github.com/guardicore/pyinstaller"}
-pyinstaller-hooks-contrib = "==2021.1" # Required to build docker with our pyinstaller branch
+pyinstaller = "==4.2"
 impacket = ">=0.9"
-importlib-metadata = "==4.0.1" # Required to build docker with our pyinstaller branch
 ipaddress = ">=1.0.23"
 netifaces = ">=0.10.9"
 odict = "==1.7.0"
+pipenv="*"
```

Comment:
This seems... weird. You already need to have pipenv installed in order to use this Pipfile.